packages/app/src/__tests__/NexusPack.test.tsx

```diff
@@ -103,22 +103,22 @@ test('show error when encountering malformed NeXus metadata', async () => {
 
   const errorSpy = mockConsoleMethod('error');
   await selectExplorerNode('nexus_malformed');
+  expect(await screen.findByText(/'signal' attribute/)).toBeVisible(); // `no_signal` is default `NXdata` child group
```

Comment:
The `resilience` and `nexus_malformed` groups now have implicit default `NXdata` children.